webapp/views/wpt-app.js

```diff
@@ -273,6 +274,16 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
     return linkedParts;
   }
 
+  handleKeyDown(e) {
+    // Ignore when something other than body has focus.
```

Comment:
Is that just the search box, or are there other things one has to avoid focusing?